static/styles/message.css

```diff
@@ -34,6 +34,31 @@ $distance_of_text_elements_from_message_box_top: 8px;
                 grid-column: 1 / 3;
                 grid-row: 1 / 2;
 
+                &.is_me_message {
+                    min-height: $first_column_width;
+                    grid-row: 1 / 3;
+                    display: grid;
+                    grid-template-columns: $first_column_width auto;
+                    grid-template-rows: auto;
+
+                    & ~ .alert-msg,
+                    & ~ .message_time,
+                    & ~ .message_controls {
+                        grid-row: 1 / 3;
+                    }
```

Comment:
I think you can squash the /me stuff into the first commit.

Reply:
Done.